### ISSUE TITLE
[crew-install]

### ISSUE THREAD

**Issue by blu** [skycocker/chromebrew#3230]
Hi,

Re-installing crew after a powerwash on an R73.0.x-beta aarch64 chromebook, I invariably get the following:

```shell
$ curl -Ls git.io/vddgY | bash
<snip>
Verifying git...
git-2.20.1-chromeos-armv7l.tar.xz: FAILED
sha256sum: WARNING: 1 computed checksum did NOT match
```

## Versions and system information
ChromeOS:
CHROMEOS_ARC_ANDROID_SDK_VERSION=25
CHROMEOS_ARC_VERSION=5376510
CHROMEOS_AUSERVER=https://tools.google.com/service/update2
CHROMEOS_BOARD_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_CANARY_APPID={90F229CE-83E2-4FAF-8479-E368A34938B1}
CHROMEOS_DEVSERVER=
CHROMEOS_RELEASE_APPID={564AC308-CBD5-485D-9EF5-EB97DBB5F264}
CHROMEOS_RELEASE_BOARD=bob-signed-mpkeys
CHROMEOS_RELEASE_BRANCH_NUMBER=104
CHROMEOS_RELEASE_BUILDER_PATH=bob-release/R73-11647.104.0
CHROMEOS_RELEASE_BUILD_NUMBER=11647
CHROMEOS_RELEASE_BUILD_TYPE=Official Build
CHROMEOS_RELEASE_CHROME_MILESTONE=73
CHROMEOS_RELEASE_DESCRIPTION=11647.104.0 (Official Build) beta-channel bob 
CHROMEOS_RELEASE_KEYSET=mp
CHROMEOS_RELEASE_NAME=Chrome OS
CHROMEOS_RELEASE_PATCH_NUMBER=0
CHROMEOS_RELEASE_TRACK=beta-channel
CHROMEOS_RELEASE_VERSION=11647.104.0
DEVICETYPE=CHROMEBOOK
GOOGLE_RELEASE=11647.104.0

Model: ASUS Chromebook Flip C101PA

Architecture: aarch64

**Comment by JL2210** [comment]
I'll leave this up to @uberhacker.

**Comment by blu** [comment]
@JL2210 This seems like a checksum issue, how is that a build type problem?

**Comment by JL2210** [comment]
The old binary was deleted from Bintray and the file downloaded on your end was probably a 404 HTML page.

**Comment by blu** [comment]
Gotcha.

**Comment by uberhacker** [comment]
The old binary is still there but has a different checksum and this was the reason I put high priority on the latest git build PR which was closed because someone here decided we needed to build it differently.  The static build of git is not necessary but some purist here thinks it is necessary. :)

**Comment by JL2210** [comment]
Do you really want to keep installing more packages in `install.sh` as git's dependency list increases?

**Comment by blu** [comment]
Guys, we could all agree we should prioritize like this:
1) make sure crew installs here and now
2) provide a better build type for git once it's ready

: )

**Comment by gavinfernandes2012** [comment]
I agree with @JL2210 that we need to lose some fat from the initial `install.sh`, it does seem a tad bit bloated, but I do also think that there's no harm having the old, non-static binary up with the right checksum so that crew can actually install as @blu said, at least until the new one's ready.

**Comment by JL2210** [comment]
The old Git build has been deleted, so please just bear with us until we
can get the new build up.

**Comment by blu** [comment]
No problem. There still a git.xz avaiable for the pertinent `ARCH` at `dl.bintray.com`, so I've installed crew with the following `install.sh` patch (aarch64-only):

```
--- /home/chronos/user/Downloads/install.sh     2019-03-28 08:00:49.029526050 +0200
+++ install.sh  2019-03-27 19:13:25.200269985 +0200
@@ -70,8 +70,8 @@
   sha256s+=('5e485a0320b298e1f5c4ff50d98c6fe6d06ad9a38d9119d580a8b469418e1e6a')
   urls+=('https://dl.bintray.com/chromebrew/chromebrew/libiconv-1.15-1-chromeos-armv7l.tar.xz')
   sha256s+=('cfa05a46d37d49479047f205e3d4ab2a8c11f953ed91e11465a1195ff2d95d01')
-  urls+=('https://dl.bintray.com/chromebrew/chromebrew/git-2.20.1-chromeos-armv7l.tar.xz')
-  sha256s+=('149a43937a9be2a8af9ebc1ab9b14fb90ff361acf1c9641959fa1c6731f842a7')
+  urls+=('https://dl.bintray.com/chromebrew/chromebrew/git-2.21.0-chromeos-armv7l.tar.xz')
+  sha256s+=('b75f477d3831241a03dc72a130d64ca1755bb0cd8d8a56fac24940de6f12ae87')
   urls+=('https://dl.bintray.com/chromebrew/chromebrew/libssh2-1.8.1-chromeos-armv7l.tar.xz')
   sha256s+=('d8fae766d77818ef95ae8a993bda0c03730d773cfb8392ceaaf6c8697206409c')
   ;;
```

**Comment by uberhacker** [comment]
Should be fixed in #3221.
